app/assets/javascripts/surveys.js

```diff
@@ -137,6 +139,43 @@
         }
       },
       {
+        identifier: 'hmrc_sept2017',
+        surveyType: 'url',
+        frequency: 6,
+        startTime: new Date('September 13, 2017').getTime(),
+        endTime: new Date('October 14, 2017 23:59:50').getTime(),
+        url: 'https://signup.take-part-in-research.service.gov.uk/?utm_campaign=' + hmrcSurveyUtmCampaignValueMap(),
```

Comment:
I think it'll be clearer to have 2 separate surveys for this one as their urls aren't really that similar.  The reason we have 1 survey for the existing hmrc one and use that `hmrcSurveyUtmCampaignValueMap` is that a) there's a lot of them, and b) their urls only differ by the value of the `utm_campaign` value, but these two a) are only 2, b) have wider differences in their urls.  Adding them to the `hmrcSurveyUtmCampaignValueMap` also means we end up changing that function - before this change the return value is a value for the `utm_campaign` param, after this change it might also be an entire url suffix including extra params, which could cause problems for call-sites that assume the return value is still just a param value.